Source/OpenTK/Platform/CoreCLR/ApplicationException.cs

```diff
@@ -0,0 +1,11 @@
+﻿#if _NET_CORECLR
+namespace System
+{
+    public class ApplicationException : Exception
```

Comment:
I went the route of putting this into "Minimal.cs", it seemed like that was where this sort of thing went (basically mock implementations of stuff for platforms that don't already have them). Doesn't really matter in the end, though, I think.